docs/source/index.rst

```diff
@@ -0,0 +1,31 @@
+.. Colibris documentation master file, created by
```

Comment:
Can't we not mix rst with md?

Reply:
Do you think it's a problem to keep them both? I would keep them both until we decide to one format.

Reply:
I think this is the point where we decide. No point in keeping them both as long as there aren't any technical constraints about them.

Reply:
The problem is that I like Markdown but I'm not sure if it's possible to have the index anything other than RST. I don't know if it's worth the effort to rewrite all documentation in RST without being sure that we want RST. I don't have a strong opinion on this issue. What's your proposal?